src/main/java/me/mrCookieSlime/Slimefun/Events/AndroidMineEvent.java

```diff
@@ -0,0 +1,44 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.block.Block;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+public class AndroidMineEvent extends Event implements Cancellable {
```

Comment:
As @WalshyDev already pointed out, a method to get the android would be nice.

And granted that this plugin has shown the middle finger to "documentation" so far, it's not a priority, but some javadocs for this class would be really awesome.